### PR TITLE
improve readability and add hover actions on concepts

### PR DESCRIPTION
**Pull request by TomGallon**

### CODE REVIEW

File: api/index.js [hunk 1]
```diff
@@ -49,10 +49,13 @@ function _getMovieById(movieId){
...
     const movie = _getMovieById(movieId);
-    const movieList = movie[listName];
-    if(movieList) {
-        console.log(movieList);
-        return movieList;
+    const peopleList = movie[listName];
+    if(peopleList) {
+        peopleList.map(people => {
+            const person = _getPerson(people.code);
+            people.linked = !!person;
```
Comment (pierr) [inline api/index.js:56]: :+1:

File: api/index.js [hunk 2]
```diff
@@ -131,7 +134,8 @@ function _getMovieResume(movieId) {
             title: movie.title,
             poster: movie.poster,
             movieType: movie.movieType,
-            productionYear: movie.productionYear
```
Comment (pierr) [inline api/index.js:134]: `{title, poster, movieType, productionYear, linked: true}`

File: app/styles/_mixins.scss [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    bottom:0;
+    padding: 10px;
+    button {
+        color: #FFF;
+        border: 2px solid #FFF;
+        margin: 0 5px;
+    }
+}
+
+@mixin linear-background() {
```
Comment (pierr) [inline app/styles/_mixins.scss:70]: colors could be params? `rgba(0,0,0,0.7), rgba(0,0,0,0.9))`

File: app/views/country/admin/country-criteria.jsx [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import React, {PropTypes} from 'react';
-import {debounce} from 'lodash/function'
+import {debounce} from 'lodash/function';
+
```
Comment (pierr) [inline app/views/country/admin/country-criteria.jsx:3]: lol

File: app/views/person/person-card-list.jsx [hunk 5]
```diff
@@ -4,27 +4,24 @@ import React, {PropTypes} from 'react';
...
-export default React.createClass({
-    displayName: 'PersonCardList',
-    propTypes: {
-        persons: PropTypes.array
-    },
+function PersonCardList({persons}) {
+    return (
+        <div data-demo='concept-card-list'>
+            {persons &&
+                persons.map(person => {
```
Comment (pierr) [inline app/views/person/person-card-list.jsx]: `persons.map(({code, leadActor, name, photoURL, role, linked}) => { ...}))`
This way you don't have to do a `const {rodrigo} = person`

**Comment by pierr** [comment]
@Tommass Great, i just add a few suggestions. 

What could be nice is a list which uses the definitions and the`textFor` helper in order to have a great example for the projects ? 

An maybe a use of `listFor`

File: app/views/country/admin/country-criteria.jsx (hunk 4 above)
Comment (TomGallon) [inline app/views/country/admin/country-criteria.jsx:3]: @pierr there's no little correction... i am so glad to correct your code :P

**Comment by TomGallon** [comment]
@pierr Thanks for your comments !

File: app/styles/_mixins.scss (hunk 3 above)
Comment (TomGallon) [inline app/styles/_mixins.scss:70]: Nope, as it white and black...

File: api/index.js (hunk 2 above)
Comment (TomGallon) [inline api/index.js:134]: @pierr i don't understand how you want me to rewrite 

``` javascript
if (movie) {
        const movieResume = {
            code: movie.code,
            title: movie.title,
            poster: movie.poster,
            movieType: movie.movieType,
            productionYear: movie.productionYear,
            linked: true
        };
        return movieResume;
    }
```

Comment (pierr) [inline api/index.js:134]: @Tommass something like

``` javascript
if (movie) {
 const {title, poster, movieType, productionYear} = movie;
return {title, poster, movieType, productionYear, linked: true}
}
```

File: app/views/country/admin/country-criteria.jsx (hunk 4 above)
Comment (pierr) [inline app/views/country/admin/country-criteria.jsx:3]: :+1:

File: api/index.js (hunk 1 above)
Comment (pierr) [inline api/index.js:56]: @Tommass  No comment on this one?

Comment (TomGallon) [inline api/index.js:56]: This code is awesome !

**Pull request merged**